jupyterhub_idle_culler/__init__.py

```diff
@@ -161,10 +161,10 @@ async def fetch_paginated(req):
     # using the `state` filter parameter. "ready" means all users who have any
     # ready servers (running, not pending).
     auth_header = {"Authorization": "token %s" % api_token}
-    resp = await fetch(HTTPRequest(url=url + "/info", headers=auth_header))
+    resp = await fetch(HTTPRequest(url=url + "/", headers=auth_header))
 
-    info = json.loads(resp.body.decode("utf8", "replace"))
-    state_filter = V(info["version"]) >= STATE_FILTER_MIN_VERSION
+    resp_model = json.loads(resp.body.decode("utf8", "replace"))
+    state_filter = V(resp_model["version"]) >= STATE_FILTER_MIN_VERSION
```

Comment:
I saw resp_model was used elsewhere as a naming convention, so I liked the idea of changing this variable name from being named `info` after switching the endpoint from `/info` to `/`.